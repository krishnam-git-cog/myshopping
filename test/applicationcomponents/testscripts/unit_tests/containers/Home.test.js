import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import Home from '../../../../../src/containers/home/Home';

describe('Home test', () => {
    it("Container - Home should match snapshot", () => {
        let shallow = createShallow();
        const component = shallow(<Home />);
        expect(component).toMatchSnapshot();
    });
});