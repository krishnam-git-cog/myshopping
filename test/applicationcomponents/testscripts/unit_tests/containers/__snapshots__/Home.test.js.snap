// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test Container - Home should match snapshot 1`] = `ShallowWrapper {}`;
